anaconda.spec.in

```diff
@@ -136,7 +136,7 @@ Requires: python3-pid
 
 # Required by the systemd service anaconda-fips.
 Requires: crypto-policies
-Requires: /usr/bin/update-crypto-policies
+Requires: crypto-policies-scripts
```

Comment:
crypto-policies-scripts is a subpackage of crypto-policies from what I see here.
https://packages.fedoraproject.org/pkgs/crypto-policies/crypto-policies-scripts/

Would not removing this line be fine? Aka the scripts subpackage would be installed by the crypto-policies.

Reply:
There's no hard dependency from crypto-policies to crypto-policies-scripts, only a Recommends.

Reply:
I see it now thanks